docs/get-started/getting-started-gcp.md

```diff
@@ -143,34 +145,18 @@ If you'd like to further accelerate init time to 4-5 minutes, create a custom Da
-to download the dataset, uncompress and 
-upload it to a GCS bucket.
+To get example data for the sample notebook, please refer to these [instructions](https://github.com/NVIDIA/spark-rapids-examples/blob/branch-22.10/docs/get-started/xgboost-examples/dataset/mortgage.md). Download the desired data, decompress it, and upload the csv files to a GCS bucket.
 
 ![Dataproc Web Interfaces](../img/GCP/dataproc-service.png)
 
-The notebook will first transcode CSV files into Parquet files and then run an ETL query to prepare
-the dataset for training.  In the sample notebook, we use 2016 data as the evaluation set and the
-rest as a training set, saving to respective GCS locations.  Using the default notebook
-configuration the first stage should take ~110 seconds (1/3 of CPU execution time with same config)
-and the second stage takes ~170 seconds (1/7 of CPU execution time with same config).  The notebook
-depends on the pre-compiled [Spark RAPIDS SQL
-plugin](https://mvnrepository.com/artifact/com.nvidia/rapids-4-spark) which is pre-downloaded by the GCP
-Dataproc [RAPIDS init
-script](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/rapids).
-
-Once data is prepared, we use the [Mortgage XGBoost4j Scala
-Notebook](../demo/GCP/mortgage-xgboost4j-gpu-scala.ipynb) in Dataproc's jupyter notebook to execute
-the training job on the GPU. Scala based XGBoost examples use [DLMC XGBoost](https://github.com/dmlc/xgboost).
-For PySpark based XGBoost, please refer to the [Spark-RAPIDS-examples 22.04 branch](https://github.com/NVIDIA/spark-rapids-examples/tree/branch-22.04)
-that uses [NVIDIA’s Spark XGBoost version](https://repo1.maven.org/maven2/com/nvidia/xgboost4j-spark_3.0/1.4.2-0.3.0/).
-Precompiled
-[XGBoost4j](https://repo1.maven.org/maven2/ml/dmlc/xgboost4j-gpu_2.12/) and [XGBoost4j
-Spark](https://repo1.maven.org/maven2/ml/dmlc/xgboost4j-gpu_2.12/) libraries can be
-downloaded from maven.  They are pre-downloaded by the GCP [RAPIDS init
-action](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/rapids).
+The sample notebook will transcode the CSV files into Parquet files before running an ETL query that prepares
+the dataset for training.  The ETL query splits the data, saving 20% of the data in a seaprate GCS location training for evaluation.  Using the default notebook configuration the first stage should take ~110 seconds (1/3 of CPU execution time with same config) and the second stage takes ~170 seconds (1/7 of CPU execution time with same config).  The notebook depends on the pre-compiled [Spark RAPIDS SQL plugin](https://mvnrepository.com/artifact/com.nvidia/rapids-4-spark) which is pre-downloaded by the GCP Dataproc [RAPIDS init script](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/rapids).
```

Comment:
@rwlee in doc, each line should be < 100 characters. Could you split the long line into many small lines? 

Same for some other long times.